sympy/vector/coordsysrect.py

```diff
@@ -45,10 +50,13 @@ def __new__(cls, name, location=None, rotation_matrix=None,
 
         """
 
+        global vector_name_list, variable_name_list, vector_alias, variable_alias
```

Comment:
is `global` really necessary? Class method should be accessible via `CoordSysCartesian.<var_name>`.